common/OpTestHost.py

```diff
@@ -423,11 +350,11 @@ def host_check_config(self, i_kernel, i_config):
     #
     def host_check_pkg_for_utility(self, i_oslevel, i_cmd):
         if 'Ubuntu' in i_oslevel:
-            l_res = self._ssh_execute("dpkg -S `which %s`" % i_cmd, timeout=60)
+            l_res = self.ssh.run_command("dpkg -S `which %s`" % i_cmd, timeout=60)
             return l_res
         else:
             l_cmd = "rpm -qf `which %s`" % i_cmd
-            l_res = self._ssh_execute(l_cmd, timeout=60)
+            l_res = self.ssh_run_command(l_cmd, timeout=60)
```

Comment:
======================================================================
ERROR: test_channel (testcases.OpTestInbandIPMI.BasicInbandIPMI)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/pridhiviraj/new_optests/stewart_patch_series/op-test-framework/testcases/OpTestInbandIPMI.py", line 56, in setUp
    self.cv_SYSTEM.load_ipmi_drivers()
  File "/home/pridhiviraj/new_optests/stewart_patch_series/op-test-framework/common/OpTestSystem.py", line 231, in load_ipmi_drivers
    l_pkg = self.cv_HOST.host_check_pkg_for_utility(l_oslevel, "ipmitool")
  File "/home/pridhiviraj/new_optests/stewart_patch_series/op-test-framework/common/OpTestHost.py", line 357, in host_check_pkg_for_utility
    l_res = self.ssh_run_command(l_cmd, timeout=60)
AttributeError: OpTestHost instance has no attribute 'ssh_run_command'

----------------------------------------------------------------------
Ran 1 test in 7.268s

FAILED (errors=1)

typo self.ssh_run_command/self.ssh.run_command